TEAM_STRUCTURES.md

```diff
@@ -194,6 +195,20 @@ The IPFS Cluster Working Group is the team implementing IPFS Cluster.
 - Design and implement IPFS Cluster.
 - Provide knowledge and APIs that organizations with large data sets can use.
 
+### Decentralized Data Stewardship
```

Comment:
> `Stewardship` the job of supervising or taking care of something, such as an organization or property.

Do we actually want to name it `Stewardship`? I'm concerned that people in the community might read it as "the IPFS WG that archives and manages datasets" in the same way that https://github.com/ipfs/archives was seen in the past as the "effort to archive and manage all the datasets on IPFS"

Reply:
I think people's confusion around IPFS repos and initiatives doesn't come down to choice of name. For example, my experience with /archives is that the README purpose isn't especially clear (to me, as an outsider), so I followed the README instructions to look at the issues to understand ongoing work. Then, my read of the issues led me to believe that /archives is *exactly* what you say it was not meant to be: PL's effort to archive datasets on IPFS. Changing the WG name only would not help clarify this misunderstanding; it is a layered communication problem.

So, I don't think naming this WG `Stewardship` will confuse people if we're clear in its home repo about scope and purpose, and make sure to edit that README as the project evolves (and when/if we hear feedback that certain terminology is unclear).

As for the selection for that word specifically, it is meant to reflect the WG's focus on understanding and supporting how people might take care of data *over time* on distributed networks, within contexts of:
- the need to manage data as mutable, connected, findable assets
- the reality that on the decentralized web, data will be managed with groups of people, and groups of organizations, both of whom will need policy layers to support human consensus on top of IPFS trustless systems

Is there another way to name the WG that you think is clearer @diasdavid ?

Reply:
Let's move forward with the _Stewardship_ word for now and change later if it becomes an issue